source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/inputhandler/CheckBoxSetInputHandler.java

```diff
@@ -7,41 +7,61 @@
 import io.adaptivecards.objectmodel.ChoiceInputVector;
 import io.adaptivecards.objectmodel.ChoiceSetInput;
 
-import java.text.ParseException;
+import java.util.Arrays;
 import java.util.List;
-import java.util.Map;
 import java.util.Vector;
 
-public class CheckBoxSetInputHandler extends BaseInputHandler
-{
-    public CheckBoxSetInputHandler(BaseInputElement baseInputElement, List<CheckBox> checkBoxList)
-    {
+public class CheckBoxSetInputHandler extends BaseInputHandler {
+    public CheckBoxSetInputHandler(BaseInputElement baseInputElement, List<CheckBox> checkBoxList) {
         super(baseInputElement);
         m_checkBoxList = checkBoxList;
     }
 
-    protected List<CheckBox> getCheckBox()
-    {
+    protected List<CheckBox> getCheckBox() {
         return m_checkBoxList;
     }
 
-    public String getInput()
-    {
+    public String getInput() {
         // no need to validate
         ChoiceSetInput choiceSetInput = (ChoiceSetInput) m_baseInputElement;
 
         Vector<String> resultList = new Vector<String>();
         ChoiceInputVector choiceInputVector = choiceSetInput.GetChoices();
-        for (int index = 0;index < m_checkBoxList.size(); index++)
-        {
-            if (m_checkBoxList.get(index).isChecked())
-            {
+        for (int index = 0; index < m_checkBoxList.size(); index++) {
```

Comment:
[nit] modify from 
for(){
}
to 
for()
{
}

Reply:
also do for if and functions

Reply:
Done